tests/functional/features/patient_access_retrieve.feature

```diff
@@ -20,7 +20,7 @@ Feature: Patient Access (Retrieve)
 		And ACCESS_DENIED is at issue[0].details.coding[0].code in the response body
 		And Patient cannot perform this action is at issue[0].details.coding[0].display in the response body
 
-	Scenario: Patient retrieve uses incorrect path
+	Scenario: Patient attempts to search for a patient
```

Comment:
This was a change made for SPINEDEM-1822. 

The test scenario although correct is not as accurate as it could be. It was attempting to search for a patient. As such, I have renamed the scenario and bundled it in with this change.